### DISCUSSION TITLE
Would Retention Tracking be possible?

### DISCUSSION THREAD

**Discussion by soraphis** [plausible/analytics#1813] [Anything Else]
On my current site I have the problem that I don't know if our visitors are customers recurring and just using the website for the "login" link to the backend or are actually new users checking out the website.

I was thinking: what if plausible would track not only a hash with a daily changing salt, but also with a 3-day-changing salt to track the 3-day retention.
 
and I was wondering if this would be possible and still be GDPR, CCPA and cookie law compliant. especially since relying on IP might not be possible for 3-day-retention and the hash might need other / more fingerprinting measures like screen resolution or such things. (to be able to stay consistent for 3 days)

**Comment by metmarkosaric** [comment]
thanks for the feedback @soraphis!

we have one new feature coming up that may help you with this. we're working on the ability to add custom dimensions to pageviews and one use case of this will be to allow you to tag "logged in" and "non logged in" traffic. then in your dashboard you'll be able to segment the stats only by those that are logged in or only by those that are logged out. the development work has already started but i cannot make any promise on when we'll be able to deliver it. you can find the details here: #320

**Comment by metmarkosaric** [comment]
We're now opened up a beta version of this to everyone. You can attach custom properties (also known as custom dimensions in Google Analytics) when sending a pageview to Plausible in order to create custom metrics such as logged in or logged out. Instructions at https://plausible.io/docs/custom-pageview-props
